src/scss/layouts/WordsOverview.scss

```diff
@@ -8,28 +6,71 @@
 
     .main_container--grid {
         margin: 30px;
+        margin-top: 0;
+        min-width: 80%;
     }
 
-        .wordBorder {
-            display: grid;
-            grid-template: 100% / repeat(3, 1fr) 1fr;
-            border: 2px solid rgb(168, 166, 163);
+    .wordBorder {
+        display: grid;
+        grid-template-columns: 1.5fr 0.5fr 1.2fr 1fr;
+        grid-template-rows: 29px;
+        border: 2px solid rgb(168, 166, 163);
+        margin: 10px;
+        padding-left: 20px;
+        box-shadow: $box-shadow;
+        background-color: $container-color;
+        height: 30px;
```

Comment:
Vyhnul bych se nastavování pevnýho height. Vždy by se to mělo odvíjet od velikosti vnitřních elementů. Představ, že něco uvnitř se zvětší a celkové výška už bude víc než 30px. Bude ti to přetékat...

Reply:
Upraveno na procenta